adapters/oidc/adapter-core/src/main/java/org/keycloak/adapters/BasicAuthRequestAuthenticator.java

```diff
@@ -71,9 +71,12 @@ public AuthOutcome authenticate(HttpFacade exchange)  {
         AccessTokenResponse atr=null;        
         try {
             String userpw=new String(Base64.decode(tokenString));
-            String[] parts=userpw.split(":");
-            
-            atr = getToken(parts[0], parts[1]);
+            int seperatorIndex = userpw.indexOf(":");
+            String user = userpw.substring(0, seperatorIndex);
+            String pw = userpw.substring(seperatorIndex + 1);
+            log.debug("user: " + user);
```

Comment:
This is not good logging. Please use a sensible message, use a single log entry and make it trace